docs/index.md

```diff
@@ -21,6 +21,7 @@ hide:
 
     <div class="ghcup-os-container" id="ghcup-instructions-unix">
       <h3>To install on Linux, macOS, FreeBSD or <a href="https://docs.microsoft.com/en-us/windows/wsl/"> WSL2</a></h3>
+      <p>You can find the system requirements for your distro on the <a href="https://www.haskell.org/ghcup/install/">installation page</a></p>
```

Comment:
```suggestion
      <p>You can find the system requirements for your distro on the <a href="https://www.haskell.org/ghcup/install/#system-requirements">installation page</a></p>
```

Reply:
jump directly to system-requirements

Reply:
I'm not sure this will look good. Please show a screenshot. This is on the main page, where we don't want to convolute the clean url box.

Reply:
This immediately moves the user to the system requirements.
It looks cool!
![image](https://github.com/user-attachments/assets/72f65fb8-988d-4ec5-bb36-955820fef957)

Reply:
I mean the index page.

Reply:
Index page view
![index_page](https://github.com/user-attachments/assets/4fd139d2-6912-4229-a9ad-18df5eb8e08b)

Reply:
Yeah, I feel it's misplaced there.

Reply:
Hm... I think it's important to see this information immediately before installation.
But I'm glad to hear what other suggestions there are where it can be placed.